cmd/utils/flags.go

```diff
@@ -998,6 +998,12 @@ var (
 		Value:  chaindatafetcher.DefaultMaxProcessingDataSize,
 		EnvVar: "KLAYTN_CHAINDATAFETCHER_MAX_PROCESSING_DATA_SIZE",
 	}
+	ChainDataFetcherRangeFetchingTracerFlag = cli.StringFlag{
+		Name:   "chaindatafetcher.rangefetching.tracer",
+		Usage:  "Chaindatafetcher range fetching tracer (callTracer or fastCallTracer)",
+		Value:  chaindatafetcher.FastCallTracer,
```

Comment:
I'm just curious. Wouldn't it be better to use integer value instead of string?

Reply:
With debug trace API, the tracer should be passed with string. This seems simpler compared to integer and converting it to string or vice versa.